services/API-service/src/api/data/data.service.ts

```diff
@@ -29,14 +27,10 @@ export class DataService {
     adminLevel: number,
     leadTime: string,
   ): Promise<GeoJson> {
-    const event = await this.getEvent(countryCode);
     let pcodes;
-    if (event) {
-      pcodes = (await this.getTriggeredAreas(event.id)).map(
-        (area): string => "'" + area.pcode + "'",
-      );
-    }
-
+    pcodes = (await this.getTriggeredAreas(countryCode)).map(
```

Comment:
`placeCodes` instead of `pcodes`
`countryCodeISO3` instead of `countryCode`
`triggeredArea` instead of `area`
`placeCode` instead of `pcode`